docs/reference/gc-toolkit/onboarding/index.md

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_position: 0
+---
+
+# Onboarding
+
+This document is your onboarding document to a fresh Guardian Connector instance.
+
+## Introduction
+
+Welcome to your Guardian Connector instance!
+
+This space is your and yours alone, maintained for you by Conservation Metrics Inc.
+
+Guardian Connector is:
+
+- where you keep your data safe (a Data Warehouse),
+- tools to get your data into it,
+- tools to use your data for your benefit.
```

Comment:
```suggestion
```

Suggest keeping this out, or linking to a different page, as we already define Guardian Connector elsewhere in the docs, and we want to keep places where we do so to a minimum (so as to avoid updating multiple definitions when it becomes outdated, etc.)